src/map.cpp

```diff
@@ -4737,6 +4737,11 @@ item map::water_from( const tripoint &p )
     }
 
     const ter_id terrain_id = g->m.ter( p );
+    if( terrain_id == t_sewage ) {
+        item ret( "sewage", 0, item::INFINITE_CHARGES );
```

Comment:
This `"sewage"` is `a sample from a treatment plant` (which provides mutations). With this change one can mutate extremely easy. I'd suggest another "sewage" which has a proper description and doesn't enable mutation.